test/browser/msw-api/regression/null-body.test.ts

```diff
@@ -1,20 +1,22 @@
 import { sleep } from '../../../support/utils'
 import { test, expect } from '../../playwright.extend'
 
-test('gracefully handles a 204 response null body during life-cycle events', async ({
-  loadExample,
-  fetch,
-  page,
-}) => {
-  await loadExample(require.resolve('./null-body.mocks.ts'))
+for (const code of [204, 205, 304]) {
+  test(`gracefully handles a ${code} response null body during life-cycle events`, async ({
+    loadExample,
+    fetch,
+    page,
+  }) => {
+    await loadExample(require.resolve('./null-body.mocks.ts'))
 
-  const errors: Array<Error> = []
-  page.on('pageerror', (pageError) => {
-    errors.push(pageError)
-  })
+    const errors: Array<Error> = []
+    page.on('pageerror', (pageError) => {
+      errors.push(pageError)
+    })
 
-  await fetch('/api/books')
-  await sleep(500)
+    await fetch(`/api/${code}`)
+    await sleep(500)
```

Comment:
Why sleep? I think this is a good opportunity to remove it. I don't believe it's needed. There are no delays in the mocked responses, and if we want to await some side effect, we need to write a proper promise for that.

Reply:
just bad copy/paste - will drop that